tests/test_comparison_validator.py

```
@@ -0,0 +1,242 @@
+    assert SYNTHESIS_PLACEHOLDER in worker.result.final_markdown
+    print("  both_valid_document_and_synthesis_error_nonfatal OK")
+
+
+# --- Regression: entfernbarer Reasoning-Vorspann leakt NICHT ins Dokument ---
+
+def test_reasoning_preamble_stripped_from_document() -> None:
+    """Gültige Analyse mit entfernbarem Reasoning-Vorspann → Doc trägt den
+    bereinigten Text, nicht den Rohtext (kein Preamble-Leak ins Dokument)."""
+    preamble = "Let me think about this carefully before I write the analysis.\n\n"
+    a_with_preamble = preamble + VALID_ANALYSIS
+    worker, events, clients = _run_worker(
```

Ruff (0.16.2)

[warning] Unpacked variable `clients` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)